pages/assets/[id].tsx

```diff
@@ -112,13 +112,12 @@ const AssetDetailView: NextPage<AssetDetailProps> = ({
   }
 
   return (
-    <>
+    <Page>
       <main>
         <Head>
           <title>{asset?.name}</title>
         </Head>
```

Comment:
i think this is a good candidate to put into `<Page />`

Reply:
That's a good idea :)

Reply:
> > didn't bother to take the fight with Typescript
> 
> what was typescript complaining about this time?

It didn't like it when I returned just the {children} and we don't need another wrapper

Reply:
oh right, because a `FunctionComponent` has to return a `JSX.Element`